components/portal/widgets/controllers.js

```diff
@@ -700,6 +706,121 @@ define(['angular', 'moment'], function(angular, moment) {
     }
   }])
 
+  // SWITCH widget type
+  .controller('SwitchWidgetController', [
+    '$scope', '$log', '$parse', 'widgetService',
+    function($scope, $log, $parse, widgetService) {
+      $scope.switching = true;
```

Comment:
Will this reliably set before [the widget-content partial reads it](https://github.com/uPortal-Project/uportal-app-framework/pull/820/files#diff-893932addcc0222b82433c64708b8c68R49) ?

Reply:
Yep. I could look it up, but even if angular did things completely backward, dom is slow, this would be initialized correctly by then.